functions/permission.js

```diff
@@ -11,5 +11,18 @@ module.exports = {
             return true;
         }
         return false;
+    },
+
+    manageReserves: (member) => {
```

Comment:
This function is great!  It'd be awesome to make the name more agnostic so that it can be used everywhere -- possibly isSuperAdmin()?